OSDOCS-17172-proxy-certificates CQA (#112906)
openshift/openshift-docs: modules/proxy-cert-expiration.adoc

> :_mod-docs-content-type: REFERENCE

Reviewer: metatdata missing. Update all modules.

Author: fixing for all files that had it missing

> :_mod-docs-content-type: REFERENCE

Reviewer: Seems more like a CONCEPT module.

Author: Done

[id="proxy-cert-expiration_{context}"]
= Proxy certificate expiration

[role="_abstract"]
The CA administrator configures the expiration term for proxy certificates before they can be used by {product-title} or {op-system}.

The user sets the expiration term of the user-provided trust bundle.

> The default expiration term is defined by the CA certificate itself. It is up to the CA administrator to configure this for the certificate before it can be used by {product-title} or {op-system}.

Reviewer: ```suggestion
The default expiration term is defined by the CA certificate itself. The CA administrator must configure the default expiration term for the certificate before the certificate can be used by {product-title} or {op-system}.
```
